sonar-plugin-api/src/main/java/org/sonar/api/batch/scm/BlameLine.java

```diff
@@ -37,11 +37,22 @@
   private Date date;
   private String revision;
   private String author;
+  
+  public BlameLine() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1186)